_pages/en/strategy-learning-automating-improving.md

```diff
@@ -8,49 +8,86 @@ sections: Quarterly Strategies
 permalink: /strategy-learning-automating-improving.html
 ---
 
-## Q1 Strategy - Continuous Improvement and Learning
+## {{ page.title }} <!-- omit in toc -->
 
 The following actions apply for the 1st quarter of the 2020-2021 fiscal year (April 2020 to June 2020).
 
-Strategies and actions will be reviewed and updated quarterly based on effectiveness and current progress towards the [Medium Term IT Picture - 2025](https://sara-sabr.github.io/ITStrategy/it-picture-medium-term.html) as well as following the [Strategy Map](https://sara-sabr.github.io/ITStrategy/strategy-summary.html) (layering on the strategies).
+- [Alignment with ESDC IT Strategy](#alignment-with-esdc-it-strategy)
+- [Guiding Policy](#guiding-policy)
+- [Expected Outcomes](#expected-outcomes)
+- [Set of actions](#set-of-actions)
+- [Measuring this strategy](#measuring-this-strategy)
+- [Appendix A - Examples of activities](#appendix-a---examples-of-activities)
+- [Appendix B - Additional References](#appendix-b---additional-references)
 
-- [Goals](#goals)
-- [Actions](#actions)
+## Alignment with ESDC IT Strategy
 
-### Goals
+This Strategy aligns with the [ESDC IT Strategy approach](esdc-it-strategy.html) and provides the means to enable the following 2 [Strategy Map](strategy-summary.html) capabilities:
 
-The goals are based on the GC IT Direction around [Digital government](https://www.canada.ca/en/government/system/digital-government.html) as well as the [DevOps Research and Assessment (DORA) - How to Transform](https://cloud.google.com/solutions/devops/devops-culture-transform) and the book explaining the findings of the State of DevOps Reports ([2019 PDF](https://services.google.com/fh/files/misc/state-of-devops-2019.pdf)), [Accelerate: The Science of Lean Software and DevOps: Building and Scaling High Performing Technology Organizations](https://books.google.ca/books?id=Kax-DwAAQBAJ&lpg=PP1&dq=Accelerate&hl=fr&pg=PP1#v=snippet&q=20%25&f=false).
+1. Learning Culture
+2. Continuous Improvement
 
-- Reduce lead time for changes and delivery of services/devices
-- Reduce frequency of incidents and errors
-- Recover faster from incidents and errors
+## Guiding Policy
 
-For more books, reports and articles supporting this strategy, you can consult our [References]({{ site.baseurl }}/references-en.html#q1-strategy---continuous-improvement-and-learning).
+IITB teams should **use up to 20% of the team's time to improve and learn**
```

Comment:
Questions: are the outcomes going to be different for each strategy? if not, does that make a difference to the above wording?  Also are we helping to establish a 'learning culture' through this strategy if the outcomes don't directly mention learning?  Also we'd have to change the wording in Appendix A if making the above change. My gut feeling is that the above language is a bit less clear than the text that was there before.  But instead of us trying to figure out what would be clear, maybe we should do some A B testing?

Reply:
Sure. Though from the engagement we've done so far there seems to be a fair amount of confusiong surrounding the use of the word learning, rather than improving (given that the reason one should learn is to be able to improve those metrics). Additionally, there is a learning team, so the vibe I get is also confusion as to why we're focused on learning while there is another team dedicated to this.

But of course, we could do some A B testing ... though not sure how we would go about doing so?

Reply:
gotcha.  This makes me want to step back a little bit.  Have we heard more of this lately and from specific people?  This feels like a perfect example of what Pia was talking about the other day re: policy and implementation needing to co-developing pieces together.  If we are 'policy' and the learning team is 'implementation', are there things we could do differently so that we are co-developing rather than simply aligning or handing policy over the wall?  So many thoughts! :)